go/internal/feast/featurestore_test.go

```diff
@@ -50,22 +49,191 @@ func TestGetOnlineFeaturesRedis(t *testing.T) {
 		},
 	}
 
-	featureViewNames := []string{"driver_hourly_stats:conv_rate",
+	featureNames := []string{"driver_hourly_stats:conv_rate",
```

Comment:
This test is currently skipped; should we unskip it if it's working locally now?

Reply:
This doesn't work locally yet, since requires registry, which apparently should be set up externally.